rocky-backend/DBtest.py

```
@@ -0,0 +1,95 @@
+test("Reject invalid ID format", res.status_code == 400)
+
+
+# update user
+if user_id:
+    res = requests.put(f"{BASE}/users/{user_id}", json={"role": "admin"})
+    test("Update user", res.status_code == 200)
+
+
+# update user bad data
+if user_id:
+    res = requests.put(f"{BASE}/users/{user_id}", json={"role": 123})
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)